codes/swift/chapter_heap/heap.swift

```diff
@@ -0,0 +1,62 @@
+/**
+ * File: heap.swift
+ * Created Time: 2024-03-17
+ * Author: nuomi1 (nuomi1@qq.com)
+ */
+
+import HeapModule
+import utils
+
+func testPush(heap: inout Heap<Int>, val: Int) {
+    heap.insert(val)
+    print("\n元素 \(val) 入堆后\n")
+    PrintUtil.printHeap(queue: heap.unordered)
+}
+
+func testPop(heap: inout Heap<Int>) {
+    let val = heap.removeMax()
+    print("\n堆顶元素 \(val) 出堆后\n")
+    PrintUtil.printHeap(queue: heap.unordered)
+}
+
+@main
+enum _Heap {
+    /* Driver Code */
+    static func main() {
+        /* 初始化堆 */
+        // Swift 的 Heap 类型同时支持最大堆和最小堆
+        var heap = Heap<Int>()
```

Comment:
We should also add the code to `heap.md` referring to other programming languages. You can add it through another PR.